flake8_quotes.py

```diff
@@ -39,14 +39,14 @@ def parse_options(cls, options):
         cls.quotes = cls.QUOTES[options.quotes]
 
     def get_file_contents(self):
-        with open(self.file, 'r') as file_to_check:
-            return file_to_check.readlines()
+        if self.filename in ('stdin', '-', None):
+            self.filename = 'stdin'
```

Comment:
Do we need to set `self.filename` here? I feel like we are contaminating the source variable =/